k2/csrc/fsa_utils_test.cu

```diff
@@ -772,8 +771,6 @@ TEST_F(StatesBatchSuiteTest, TestBackwardScores) {
               GetBackwardScores<float>(cpu_fsa_vec, nullptr, false);
           CheckArrayData(scores, cpu_scores);
         }
-        // TODO(haowen): will get -nan in below tests, may be related to use
```

Comment:
Ignore this, as I have fixed this issue in the last PR but forgot to remove this comment. (we get -nan because we may add `inf` with `-inf` before.)